src/handlers/processors.ts

```diff
@@ -8,10 +8,11 @@ import { checkPullRequests } from "./assign/auto";
 import { createDevPoolPR } from "./pull-request";
 import { runOnPush } from "./push";
 import { incentivizeComments, incentivizeCreatorComment } from "./payout";
+import { findDuplicateOne } from "./issue";
 
 export const processors: Record<string, Handler> = {
   [GithubEvent.ISSUES_OPENED]: {
-    pre: [nullHandler],
+    pre: [findDuplicateOne],
```

Comment:
@0xcodercrane do you think it makes sense that it is in the `GithubEvent.ISSUES_OPENED[pre]` hook instead of adding it into the `GithubEvent.ISSUES_OPENED[action]` array?

Reply:
@0xcodercrane rfc

Reply:
@pavlovcik is right. Let me push and merge the PR

Reply:
https://github.com/ubiquity/ubiquibot/pull/608/commits/89e670562e89856c6cb8607cd6a39696f8bfeffa